pkgs/development/compilers/ispc/default.nix

```diff
@@ -42,25 +40,26 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin
-    cp ispc $out/bin
+    cp bin/ispc $out/bin
   '';
 
   checkPhase = ''
-    export ISPC_HOME=$PWD
+    export ISPC_HOME=$PWD/bin
     for target in $testedTargets
     do
       echo "Testing target $target"
       echo "================================"
       echo
+      cd ../
```

Comment:
you may want to restore the PWD after this checkPhase, or else it puts me into an invalid directory for the install phase
```
installing
install flags: SHELL=/nix/store/l6h4ya0wzb4b8mr0y58k2gh2nhfql4sn-bash-4.4-p23/bin/bash install
ldconfig: Can't open cache file /nix/store/6yaj6n8l925xxfbcd65gzqx3dz7idrnn-glibc-2.27/etc/ld.so.cache
: No such file or directory
 ============================== !!! WARNING !!! ===============================
 Location of LLVM files in your PATH is different than path in LLVM_HOME
 variable (or LLVM_HOME is not set). The most likely this means that you are
 using default LLVM installation on your system, which is very bad sign.
 Note, that ISPC uses LLVM optimizer and is highly dependent on it. We recommend
 using *patched* version of LLVM 3.8. Patches are availible in
 llvm_patches folder. You can build LLVM manually, or run our scripts, which
 will do all the work for you. Do the following:
 1. Create a folder, where LLVM will reside and set LLVM_HOME variable to its
   path.
 2. Set ISPC_HOME variable to your ISPC location (probably current folder).
 3. Run alloy.py tool to checkout and build LLVM:
   alloy.py -b --version=3.8
 4. Add $LLVM_HOME/bin-3.8/bin path to your PATH.
 ==============================================================================
make: *** No rule to make target 'install'.  Stop.
```

Reply:
Hm, I thought that would run in a subshell.  Guess I'm forgetting my shell script trivia.